nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/StandardNiFiServiceFacade.java

```diff
@@ -1574,7 +1574,7 @@ public ProcessGroupEntity createArchive() {
     }
 
     @Override
-    public ProcessorEntity setProcessorAnnotationData(final Revision revision, final String processorId, final String annotationData) {
```

Comment:
Originally pulled over the override method updateProcessor however forgot that you mentioned groupdId is not required in this case. Updated PR has the use of the existing updateProcessor.